PHPhomework4

```diff
@@ -0,0 +1,17 @@
+<?php
+function findPrimeNumberBetween($n)
+{
+  for ($m = 2; $m <= $n; $m++) {
+  $x = 0;
+  $y = 1;
+  for ($i = 2; $i < $m ; $i++) {
+   $x = $m % $i;
+   $y *= $x;
+  }
+  if($y != 0) {
+   echo "$m"."<br>";
+  }
+  }
+}
```

Comment:
indent 1 tab
fix same as above

Reply:
I think you should configure 1 tab = 4 spaces